tests/test_sensor.py

```diff
@@ -375,6 +377,7 @@ async def async_test_em_rms_voltage(
     zha_gateway: Gateway, cluster: Cluster, entity: PlatformEntity
 ) -> None:
     """Test electrical measurement RMS Voltage sensor."""
+    assert entity.extra_state_attribute_names == {"measurement_type", "rms_voltage_max"}
```

Comment:
We never checked if the behavior actually correctly include the "max extra state attributes" before, so also added this.